src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/mpack/addin.info

```diff
@@ -17,6 +17,7 @@
     <Import assembly="Microsoft.AspNetCore.Razor.LanguageServer.dll" />
     <Import assembly="Microsoft.AspNetCore.Razor.LanguageServer.Common.dll" />
     <Import assembly="Microsoft.AspNetCore.Razor.LanguageServer.Protocol.dll" />
+    <Import assembly="Microsoft.Extensions.ObjectPool.dll" />
```

Comment:
There is also an xml file in the Properties folder that probably should be updated.

Reply:
Thanks!